src/Microsoft.ML.Dnn/ImageClassificationTransform.cs

```diff
@@ -674,7 +688,8 @@ private void VariableSummaries(RefVariable var)
                         bottleneckTensor,
                         shape: bottleneckTensorDims,
                         name: "BottleneckInputPlaceholder");
-                    _learningRateInput = tf.placeholder(tf.float64, null, name: "learningRateInputPlaceholder");
+                    if(useLearningRateScheduler)
+                        _learningRateInput = tf.placeholder(tf.float64, null, name: "learningRateInputPlaceholder");
```

Comment:
I think we should use `tf.float32`.

Reply:
Thanks for the suggestion. I realize we don't need a high precision to store learning rate. Will change that.